src/main/kotlin/com/exactpro/th2/lwdataprovider/configuration/Configuration.kt

```diff
@@ -32,10 +33,14 @@ class CustomConfigurationClass(
     val responseQueueSize: Int? = null,
     val execThreadPoolSize: Int? = null,
     val convThreadPoolSize: Int? = null,
+    @Deprecated("use parameter batchSizeBytes instead to set limit in bytes. " +
+            "This parameter does not have any effect anymore. " +
```

Comment:
Maybe remove it in this case. Custom config should include only effective settings

Reply:
This will break the configuration serialization if this parameter is set (because of an unknown parameter). I don't think we should remove it right now. Maybe later

Reply:
I don't think so, because user doesn't look an the code or readme when he/she will updates component

Reply:
A user doesn't read the exceptions in logs either) User doesn't read anything at all) So why bother the user with exception during deserislization

Reply:
User will faced to component which restarts when specify unknown option.

Reply:
And the user will go to us to fix it because they don't read error messages)
However, I am tired of arguing. I will remove both unused properties